### DISCUSSION TITLE
Does tsup respect the exports attribute in package.json for determining ESM/CJS outputs?

### DISCUSSION THREAD

**Discussion by sgarfinkel** [egoist/tsup#732] [Q&A]
I only see reference to the `module` and `main` fields. Using conditional exports is now the recommended approach: https://nodejs.org/api/packages.html#conditional-exports so I was just curious if this is a supported method of defining expected outputs.

**Comment by taylor-wood-morpheus** [comment]
This is really important!

Esbuild has already added a `--conditions` flag, which tsup could leverage.
